tests/integration/test_installation.py

```diff
@@ -166,11 +168,18 @@ def test_running_real_remove_backup_groups_job(ws, installation_ctx):
 
     installation_ctx.deployed_workflows.run_workflow("remove-workspace-local-backup-groups")
 
-    @retried(on=[NotFound], timeout=timedelta(seconds=120))
```

Comment:
The opposite of the `retried` should be done: the test should pass on a `NotFound` and retried on no `NotFound` until timeout

Reply:
@JCZuurmond why not retrying on a different exception? the following is still a smaller change.

```
    @retried(on=[AssertionError], timeout=timedelta(seconds=120))
    def wait():
        ws.groups.get(ws_group_a.id)
        assert False

    wait()
```

Reply:
this does the trick